config/config.go

```diff
@@ -72,19 +72,21 @@ type Config struct {
 	// for client this means the hostname or IP address of the local server
 	Localhost string
 
-	// TLS cert is used when TLS conn pool is used as transportation layer
+	// WHInterconnectTLS cert is used when TLS conn pool is used as transportation layer
```

Comment:
I can see how you wanted to differentiate the tunnel ENVs from wh-client<->local-endpoint ENVs, but it seems to overly complicate things (from the perspective of the user who will be using the client). Unless you feel strongly about this, I'll leave `TLS`, `TLSCert` and `Insecure` as is.

Reply:
Gotcha, yeah wasn't considering the perspective of the user there. It'll make our dev lives potentially more difficult. But from a user perspective it's certainly better convention